### ISSUE TITLE
Is it fully compatible with 1.5.x?

### ISSUE THREAD

**Issue by stuffi3000** [LuxInno/ValidatorHOI4#2]
Everything is in the title :p

**Comment by rweber89** [comment]
I believe there might be small inconsistencies related to the Waking the Tiger DLC - concerning unit declaration files. Other than that everything is working as before.

If you notice anything just contact me here.

**Comment by stuffi3000** [comment]
I found an issue:
Inside a Unit file, when a name is defined by the follwing:
`division_name = {
			is_name_ordered = yes
			name_order = 1
		}`
Then the Validator shows it as "DIVISION_NAME_MISSING".
![Screenshot](https://image.ibb.co/j0weS8/Unbenannt.png)

**Comment by rweber89** [comment]
@stuffi3000 Yes I believe they changed the whole division definition thing in Waking the Tiger. I have not yet had the time to adapt the validator to the changes. I will let you know when I do so!
